.github/workflows/on-push-lint-charts.yml

```diff
@@ -0,0 +1,57 @@
+name: Lint and Test Charts
+
+on: 
+  push:
+    paths:
+      - 'charts/**' 
+      - '.github/**'            
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0        
+          
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      - uses: actions/setup-python@v2
```

Comment:
What is the reason to install python?

Reply:
I don't know :) The chart-testing action seems to require it, it's part of the default example at https://github.com/helm/chart-testing-action

Reply:
Could you add a comment that this is a requirement for the chart testing action. They use it for `yamllint` (and some more): Python

Reply:
Added, and shuffled the order of the steps to make this clearer